test/jsonInt64.spec.js

```diff
@@ -0,0 +1,22 @@
+var mocha  = require('mocha');
+var assert = require('chai').assert;
+var expect = require('chai').expect;
+var Int64 = require('node-int64');
+var JSONInt64 = require('../index');
+
+describe("Tests the Int64 support", function(){
+    var input = '{"int64MaxValue":9223372036854775807,"int64MinValue":-9223372036854775807,"number":123}';
```

Comment:
can't we use the ``Number`` constants I pointed out before?

Reply:
These are outside of safe integer range